airflow/utils/dag_processing.py

```diff
@@ -559,7 +560,7 @@ def __init__(self,
                      AbstractDagFileProcessorProcess
                  ],
                  processor_timeout: timedelta,
-                 signal_conn: multiprocessing.connection.Connection,
+                 signal_conn: Connection,
```

Comment:
```
2020-05-04T07:45:39.6352361Z [1BTraceback (most recent call last):
2020-05-04T07:45:39.6354423Z   File "/usr/local/bin/airflow", line 11, in <module>
2020-05-04T07:45:39.6358388Z     load_entry_point('apache-airflow', 'console_scripts', 'airflow')()
2020-05-04T07:45:39.6359382Z   File "/opt/airflow/airflow/__main__.py", line 40, in main
2020-05-04T07:45:39.6359738Z     args.func(args)
2020-05-04T07:45:39.6360081Z   File "/opt/airflow/airflow/cli/cli_parser.py", line 51, in command
2020-05-04T07:45:39.6360410Z     func = import_string(import_path)
2020-05-04T07:45:39.6360763Z   File "/opt/airflow/airflow/utils/module_loading.py", line 32, in import_string
2020-05-04T07:45:39.6361068Z     module = import_module(module_path)
2020-05-04T07:45:39.6361423Z   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
2020-05-04T07:45:39.6361760Z     return _bootstrap._gcd_import(name[level:], package, level)
2020-05-04T07:45:39.6362144Z   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
2020-05-04T07:45:39.6364835Z   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
2020-05-04T07:45:39.6365334Z   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
2020-05-04T07:45:39.6365765Z   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
2020-05-04T07:45:39.6366126Z   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
2020-05-04T07:45:39.6366495Z   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
2020-05-04T07:45:39.6366853Z   File "/opt/airflow/airflow/cli/commands/db_command.py", line 24, in <module>
2020-05-04T07:45:39.6369304Z     from airflow.utils import cli as cli_utils, db
2020-05-04T07:45:39.6371950Z   File "/opt/airflow/airflow/utils/db.py", line 27, in <module>
2020-05-04T07:45:39.6372505Z     from airflow.jobs.base_job import BaseJob  # noqa: F401 # pylint: disable=unused-import
2020-05-04T07:45:39.6372688Z   File "/opt/airflow/airflow/jobs/__init__.py", line 22, in <module>
2020-05-04T07:45:39.6372842Z     import airflow.jobs.scheduler_job  # noqa
2020-05-04T07:45:39.6373000Z   File "/opt/airflow/airflow/jobs/scheduler_job.py", line 51, in <module>
2020-05-04T07:45:39.6373136Z     from airflow.utils.dag_processing import (
2020-05-04T07:45:39.6373467Z   File "/opt/airflow/airflow/utils/dag_processing.py", line 529, in <module>
2020-05-04T07:45:39.6376131Z     class DagFileProcessorManager(LoggingMixin):  # pylint: disable=too-many-instance-attributes
2020-05-04T07:45:39.6376656Z   File "/opt/airflow/airflow/utils/dag_processing.py", line 563, in DagFileProcessorManager
2020-05-04T07:45:39.6377088Z     async_mode: bool = True):
2020-05-04T07:45:39.6377555Z AttributeError: module 'multiprocessing' has no attribute 'connection'
```
I don't know what happened here, but without this change I have the above exception.